MangoPay/PayOut.php

```diff
@@ -7,6 +7,20 @@
  */
 class PayOut extends Transaction
 {
+    /**
+     * The unique identifier of the bank account.
+     *
+     * @var string
+     */
+    public $BankAccountId;
```

Comment:
What is the reason for adding this to the PayOut class?

Reply:
This property was identified in both the documentation and PHP SDK. I verified its existence through debug logs and implemented it accordingly.